qiita_ware/ebi.py

```diff
@@ -885,8 +866,14 @@ def generate_demultiplexed_fastq(self, rewrite_fastq=False, mtime=None):
         """
         ppd = self.preprocessed_data
 
-        if not isdir(self.ebi_dir) or rewrite_fastq:
-            makedirs(self.ebi_dir)
+        dir_not_exists = not isdir(self.full_ebi_dir)
+        if dir_not_exists or rewrite_fastq:
+            if dir_not_exists:
+                makedirs(self.full_ebi_dir)
+                filetype_id = convert_to_id('directory', 'filepath_type')
+                insert_filepaths([(self.full_ebi_dir, filetype_id)], -1,
```

Comment:
Thought about it but I'm not sure we want this because this will imply that we will need to test which filepaths are folders and which ones are files for display, which will complicate code/functionality. I think we could add this later if we see the necessity of displaying this files in the GUI. What do you think?